app/subscriber.py

```diff
@@ -20,7 +20,7 @@
 
 def receipt_to_case(message: Message):
     """
-    Callback for handling new pubsub messages which attempts to publish a receipt to the case service
+    Callback for handling new pubsub messages which attempts to publish a receipt or refusal to the events exchange
```

Comment:
I think refusals come through either Contact Centre or FWMT via the events exchange, and not through here?

Reply:
good spot